docs/CaliptraHardwareSpecification.md

```diff
@@ -18,6 +19,19 @@ This document provides definitions and requirements for a Caliptra cryptographic
 
 For information on the Caliptra Core, see the [High level architecture](https://chipsalliance.github.io/Caliptra/doc/Caliptra.html#high-level-architecture) section of [Caliptra: A Datacenter System on a Chip (SoC) Root of Trust (RoT)](https://chipsalliance.github.io/Caliptra/doc/Caliptra.html).
 
+## Key Caliptra Core 2.0 Changes
+* AXI subordinate replaces APB interface of Caliptra 1.x hardware
+* SHA Accelerator functionality now available exclusively over mailbox
+    * SHA Accelerator adds new SHA save/restore functionality
```

Comment:
Should we wait to list save/restore function for SHA Accelerator? It is not yet enabled.

Reply:
Maybe just a separate bullet listing it for the sha512 block instead

Reply:
just a spec; PR will come through as a part of Freeze, Mojtaba is there